vj4/model/document.py

```diff
@@ -324,17 +325,21 @@ def get_multi_status(*, fields=None, **kwargs):
   return doc
 
 
-async def rev_set_status(domain_id, doc_type, doc_id, uid, rev, **kwargs):
+async def rev_set_status(domain_id, doc_type, doc_id, uid, rev, return_doc=True, **kwargs):
```

Comment:
Why?

Reply:
In `recalc_contest_status`, `rev_set_status` don't need to return docs.